### PR TITLE
Need to update HorizontalPodAutoscaler yaml file under section Autoscaling for Memory Utilization 4.x

### PR DESCRIPTION
**Pull request by mburke5678**
Applying similar changes to 4.x as reported in https://bugzilla.redhat.com/show_bug.cgi?id=1801532

3.11 PR https://github.com/openshift/openshift-docs/pull/19761

### CODE REVIEW

**Comment by mburke5678** [comment]
@weinliu Please take a look.

**Comment by mburke5678** [comment]
@weinliu Please take a look. The topic in 4.x is incorrect and I would like to make it right.

**Comment by mburke5678** [comment]
@weinliu Please take a look.

**Comment by mburke5678** [comment]
@weinliu Please take a look.

File: modules/nodes-pods-autoscaling-creating-memory.adoc [hunk 1]
```diff
@@ -30,101 +41,116 @@ You can use the `oc describe PodMetrics <pod-name>` command to determine if metr
...
+<5> Specify the name of the object to scale.
+<6> Specify the minimum number of replicas when scaling down.
+<7> Specify the maximum number of replicas when scaling up.
+<8> Use the `metrics` parameter for memory utilization.
+<9> Specify `memory` for memory utilization.
+<10> Set to `Utilization`.
+<11> The type of must be either `averageUtilization` or `averageValue`.
 
 . Create the autoscaler from the above file:
 +
```
[the remark this answers is not in the document]
Comment (weinliu) [reply modules/nodes-pods-autoscaling-creating-memory.adoc]: @mburke5678 ^^

File: modules/nodes-pods-autoscaling-creating-memory.adoc [hunk 2]
```diff
@@ -30,101 +41,116 @@ You can use the `oc describe PodMetrics <pod-name>` command to determine if metr
...
-Window:                1m0s
+  Creation Timestamp:  2020-02-14T22:21:14Z
+  Self Link:           /apis/metrics.k8s.io/v1beta1/namespaces/openshift-kube-scheduler/pods/openshift-kube-scheduler-ip-10-0-129-223.compute.internal
+Timestamp:             2020-02-14T22:21:14Z
+Window:                5m0s
 Events:                <none>
+
 ----
 
 .Procedure
```
[the remark this answers is not in the document]
Comment (weinliu) [reply modules/nodes-pods-autoscaling-creating-memory.adoc]: @mburke5678 ^^

**Comment by weinliu** [comment]
> @weinliu Please take a look.

I seems my review comment did not get displayed untill  I click "Submitt", @mburke5678 can you seem my comment above? Thanks.

**Comment by mburke5678** [comment]
@weinliu Thank you for your comments. I have made the changes as you suggested.

File: modules/nodes-pods-autoscaling-creating-cpu.adoc [hunk 3]
```diff
@@ -6,8 +6,13 @@
...
 = Creating a horizontal pod autoscaler for CPU utilization
 
-You can create a horizontal pod autoscaler (HPA) to automatically scale pods when CPU usage exceeds a specified percentage.
-You create the HPA for a replication controller or deployment controller, based on how your pods were created.
+You can create a horizontal pod autoscaler (HPA) for an existing DeploymentConfig or ReplicationController object
+that automatically scales the Pods associated with that object in order to maintain the CPU usage you specify.
+
+The HPA increases and decreases the number of replicas between the minimum and maximum numbers to maintain the specified CPU utilization across all Pods.
+
+When autoscaling for CPU utilization, you can use the `oc autoscale` command and specify the minumim and maximum number of Pods you want to run at any given time and the average CPU utilization your Pods should target. If you do not specify a minimum, the Pods are given default values from the {product-title} server.
```
Comment (ahardin-rh) [inline modules/nodes-pods-autoscaling-creating-cpu.adoc]: s/minumim/minimum

File: modules/nodes-pods-autoscaling-creating-cpu.adoc [hunk 4]
```diff
@@ -45,40 +50,141 @@ Events:                <none>
...
-$ oc get dc example
-NAME      REVISION   DESIRED   CURRENT   TRIGGERED BY
-example   1          3         3         config
+$ oc get hpa hpa-resource-metrics-memory
+
+NAME                                     REFERENCE                       TARGETS         MINPODS   MAXPODS   REPLICAS   AGE
+oc get hpa hpa-resource-metrics-memory   ReplicationController/example   2441216/500Mi   1         10        1          20m
+----
+
+For example, the following command creates a horizontal pod autoscaler that maintains between 3 and 7 replicas of the Pods that are controlled by the `image-registry` DeploymentConfig in order to maintain an average CPU utilization of 75% across all Pods.
```
Comment (ahardin-rh) [inline modules/nodes-pods-autoscaling-creating-cpu.adoc]: s/pod/Pod

File: modules/nodes-pods-autoscaling-creating-cpu.adoc [hunk 5]
```diff
@@ -45,40 +50,141 @@ Events:                <none>
...
+
+----
+$ oc autoscale dc/image-registry --min 3 --max 7 --cpu-percent=75
+deploymentconfig "image-registry" autoscaled
+----
+
+The command creates a horizontal pod autoscaler with the following definition:
```
Comment (ahardin-rh) [inline modules/nodes-pods-autoscaling-creating-cpu.adoc]: s/pod/Pod

File: modules/nodes-pods-autoscaling-creating-memory.adoc [hunk 6]
```diff
@@ -6,7 +6,15 @@
 
 = Creating a horizontal pod autoscaler object for memory utilization
 
-You can create a horizontal pod autoscaler to automatically scale pods in a Deployment when memory usage exceeds a specified limit.
+You can create a horizontal pod autoscaler (HPA) for an existing DeploymentConfig or ReplicationController object
+that automatically scales the Pods associated with that object in order to maintain the average memory utilization you specify, either a direct value or 
+a percentage of requested memory.
```
Comment (ahardin-rh) [inline modules/nodes-pods-autoscaling-creating-memory.adoc]: strange line break after "or"

**Comment by ahardin-rh** [comment]
@mburke5678 Just some minor comments from me. Looks great!

**Comment by mburke5678** [comment]
@weinliu Do the changes I made based on your comments look OK?

**Comment by weinliu** [comment]
> @weinliu Do the changes I made based on your comments look OK?

@mburke5678 LGTM. Thanks!

**Pull request merged**